docs/querying/functions.md

```diff
@@ -464,6 +464,89 @@ by the number of seconds under the specified time range window, and should be
 used primarily for human readability.  Use `rate` in recording rules so that
 increases are tracked consistently on a per-second basis.
 
+## `info()`
```

Comment:
```suggestion
## `info()` (experimental)
```
 
 Perhaps something to add to all experimental functions?

Reply:
> Perhaps something to add to all experimental functions?

Could be, not sure :)